tests/utils_tests/test_functional.py

```diff
@@ -130,3 +131,11 @@ def test_lazy_equality(self):
 
         self.assertEqual(lazy_a(), lazy_b())
         self.assertNotEqual(lazy_b(), lazy_c())
+
+    def test_lazy_repr(self):
+        """
+        Tests that a proxy of lazy object's repr equals to the object's value
```

Comment:
Yes I understand, any idea of how to try only the lazy method individually? Tried with lambdas but couldn't make it work properly.